src/wave.js

```diff
@@ -71,6 +72,15 @@ class WaveBase {
         this.renderer.domElement.style.height = "100%";
         this.container.appendChild(this.renderer.domElement);
         this.renderer.setSize(this.WIDTH, this.HEIGHT);
+
+        // for atom labels
+				this.labelRenderer = new CSS2DRenderer();
```

Comment:
Remove extra indent

Reply:
Same as https://github.com/Exabyte-io/wave.js/pull/38/files#r331610274 - calling this `rendererCSS2D` might be more intuitive